File: solutions/2010-check-if-word-equals-summation-of-two-words/solution.py
class Solution(object):
    def isSumEqual(self, firstWord, secondWord, targetWord):
        """
        :type firstWord: str
        :type secondWord: str
        :type targetWord: str
        :rtype: bool
        """
        ls="".join([str(ord(i)-97) for i in firstWord]).strip()
        ls1="".join([str(ord(i)-97) for i in secondWord]).strip()
        ls2="".join([str(ord(i)-97) for i in targetWord]).strip()
        return int(ls)+int(ls1)==int(ls2)

File: solutions/2010-check-if-word-equals-summation-of-two-words/solution.java
class Solution {
    public boolean isSumEqual(String firstWord, String secondWord, String targetWord) {
        String a="",b="",c="";
        for(char s:firstWord.toCharArray())
            a+=s-97;
        for(char s:secondWord.toCharArray())
            b+=s-97;
        for(char s:targetWord.toCharArray())
            c+=s-97;
        return (Integer.parseInt(a)+Integer.parseInt(b))==Integer.parseInt(c);
    }
}
